genie-web/src/test/groovy/com/netflix/genie/web/tasks/job/JobCompletionServiceSpec.groovy

```diff
@@ -224,4 +261,50 @@ class JobCompletionServiceSpec extends Specification {
             assert !d.exists()
         })
     }
+
+    /**
+     * Verify the presence of failure tags
+     *
+     * @param tags tags to be verified
+     * @param throwable exception thrown on failure
+     * @return true if all failure tags exist
+     */
+    private Boolean verifyFailureTags(Map<String, String> tags,
+                                      final Throwable throwable) {
+        final Map<String, String> failureTags = MetricsUtils.newFailureTagsMapForException(throwable);
+
+        return tags.get(MetricsConstants.TagKeys.STATUS)
+                .equals(failureTags.get(MetricsConstants.TagKeys.STATUS)) \
```

Comment:
Why create a whole new map just to check the value of a key matches, as opposed to just use the constant that you expect?